portfolio/src/main/java/com/google/sps/servlets/ListImagesServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import com.google.api.gax.paging.Page;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Shows all of the images uploaded to Cloud Storage. */
+@WebServlet("/list-images")
+public class ListImagesServlet extends HttpServlet {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
+
+    @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // List all of the uploaded files.
+    String projectId = "rshillingford-sps-spring21";
```

Comment:
If these variables(projectId, bucketName) are not changing throughout your file & in new methods you might create, you might want to consider making them static final like the serialVersionUID